File: src/main/Python/boj_step10/boj14215.py
import sys
cin = sys.stdin.readline

a, b, c = map(int, cin().split())
M = max(a, max(b, c))
if M * 2 >= a + b + c: print((a + b + c - M) * 2 - 1)
else: print(a + b + c)

File: src/main/Cpp/boj_step10/boj14215.cpp
#include <iostream>
using namespace std;

int main() {
    int a, b, c;
    cin >> a >> b >> c;
    int max_value = max(a, max(b, c));

    if (max_value * 2 >= a + b + c)
        cout << (a + b + c - max_value) * 2 - 1;

    else
        cout << a + b + c;
}